music.rb

```diff
@@ -0,0 +1,15 @@
+equire_relative 'item'
```

Comment:
Kindly require this to be require_relative instead of equire_relative inorder to be able to access the item class